scripts/colab_fuzzy_dedup_study.py

```
@@ -0,0 +1,458 @@
+
+def parse_str_values(spec: str) -> list[str]:
+    vals = [part.strip() for part in spec.split(",") if part.strip()]
+    if not vals:
+        raise ValueError("No values parsed.")
+    return vals
+
+
+def ensure_nemo_deps() -> None:
+    missing = []
+    try:
+        import torch  # noqa: F401
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)